src/app/exercise/editors/editors.component.ts

```diff
@@ -37,10 +60,8 @@ export class EditorsComponent {
   }
 
   isOpenFile(file) {
-    if (!file || !this.currentFile) {
-      return;
-    }
-    return file.path === this.currentFile.path;
+    if (!file) { return; }
```

Comment:
I actually think I can remove this check. There's never a chance that this function is called without the file